src/Module.Server/Common/Models/CrpgAgentStatCalculateModel.cs

```diff
@@ -448,7 +448,7 @@ private float ImpactofStrAndWeaponLengthOnTimeToMaxSpeed(int weaponLength, int s
 
     private int MaxWeaponLengthForStrLevel(int strengthSkill)
     {
-        int uncappedMaxWeaponLength = (int)(45 + (strengthSkill - 3) * 9 + Math.Pow((strengthSkill - 3) * 0.14677993f, 12));
+        int uncappedMaxWeaponLength = (int)(45 + (strengthSkill - 3) * 9 + Math.Pow(Math.Min(strengthSkill - 3, 24) * 0.14677993f, 12));

```

Comment:
Does that need to updated in the UI?

Reply:
no.
![image](https://github.com/verdie-g/crpg/assets/47090987/0a558921-e14a-4f3a-9f62-44b0f50cb3a5)
this is with 54 str

Reply:
Even if js has a dumb behavior for overflows it's better to keep the code aligned. I'll do the commit.